src/views/validated-designs/guide/index.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Head from 'next/head'
+import BaseLayout from 'layouts/base-layout'
+import MobileMenuLevelsGeneric from 'components/mobile-menu-levels-generic'
+import InlineLink from 'components/inline-link'
+
+export default function ValidatedDesignGuideView() {
+	return (
+		<BaseLayout mobileMenuSlot={<MobileMenuLevelsGeneric />}>
```

Comment:
Added `BaseLayout` here so it doesn't look totally odd.
<img width="1083" alt="Screenshot 2023-11-13 at 4 50 42 PM" src="https://github.com/hashicorp/dev-portal/assets/36613477/11ec631f-0241-445d-8456-fd30fede2295">